tsl/test/sql/continuous_aggs_usage.sql

```diff
@@ -202,8 +188,8 @@ SELECT * FROM device_readings_mat_only ORDER BY time_bucket;
 -- jit aggregate should have 4 rows
 SELECT * FROM device_readings_jit ORDER BY time_bucket;
 
-REFRESH MATERIALIZED VIEW device_readings_mat_only;
-REFRESH MATERIALIZED VIEW device_readings_jit;
+CALL refresh_continuous_aggregate('device_readings_mat_only', NULL, 30);
+CALL refresh_continuous_aggregate('device_readings_jit', NULL, 30);
```

Comment:
You need to look at the output log to see that this makes sense. This is the value that the previous materializer stopped at so it makes sense to stop here to minimize test output differences.

The next refresh actually ended at 70, but was the last one and covered all data so it can be replaced with an infinite refresh.

Reply:
Could you add a comment with an explanation in the SQL file so that it is clear that it is intentional?

Reply:
I will, but why isn't it clear that it is intentional if the value is there?

Reply:
Compare the following reasoning: 

> I put the limit here since I know that there is nothing more to update and even though I have a limit everything will be updated.

With

> I put a limit here because I do not want to update everything and it is important that the entire range is not updated.

If I get here later and want to update the tests, I have no idea why you have a limit there and it will require some work to figure out if the specific value is important or not and if I have a lot of diffs in the tests as a result of a change, it will be tricky to figure out what is relevant and what is not relevant.